src/service/execution/test.execution.service.ts

```diff
@@ -54,4 +55,14 @@ export class DefaultTestExecutionService extends TestExecutionService {
     return `${this.serviceUrl}${urlPath}?resource=${encodedPath}`;
   }
 
+  private toElementState(state: string): ElementState {
```

Comment:
… if `ElementState` is renamed (see above), this should be renamed, accordingly.